nodebuilder/blob/cmd/blob.go

```diff
@@ -41,15 +41,18 @@ func init() {
 		&fee,
 		"fee",
 		-1,
-		"specifies fee for blob submission",
+		"specifies fee (in TIA) for blob submission [optional]",
 	)
 
 	submitCmd.PersistentFlags().Uint64Var(
 		&gasLimit,
 		"gas.limit",
 		0,
-		"specifies max gas for the blob submission",
+		"sets the maximum amount of gas that is allowed to consume during blob submission [optional]",
```

Comment:
Gas is always fully consumed with no refund.
```suggestion
		"sets the amount of gas that is consumed during blob submission [optional]",
```